docs/vhost-user-net-testing.md

```diff
@@ -0,0 +1,126 @@
+# Vhost-user-net
```

Comment:
Need a better title

Reply:
Change it to be "How to test vhost-user net with OpenVSwitch/DPDK".